emcc.py

```diff
@@ -1577,8 +1577,11 @@ def get_final():
 
     with ToolchainProfiler.profile_block('source transforms'):
       # Embed and preload files
-      if shared.Settings.SPLIT_MEMORY:
-        options.no_heap_copy = True # copying into the heap is risky when split - the chunks might be too small for the file package!
+      # copying into the heap is risky when split - the chunks might be too small for the file package!
+      # Also, MEMFS is not aware of heap resizing feature in wasm, so if MEMFS and Wasm memory growth are used together, force
+      # no_heap_copy to be enabled.
+      if shared.Settings.SPLIT_MEMORY or (shared.Settings.BINARYEN and shared.Settings.ALLOW_MEMORY_GROWTH):
+        options.no_heap_copy = True
```

Comment:
perhaps we should log out a warning when we set no heap copy?

Reply:
I don't want to print a warning, because people might assume that --no-heap-copy might be somehow a bad option in general to use. Added a `logging.info` instead, how's that?

Reply:
Hmm, when is `info` shown? I know a warning always is and a debug is shown when `EMCC_DEBUG=1` is in the env (which seems ok for this), but in what situation would a user see `info`?

Reply:
`info` is also shown always, but just without the yellow color and the word `warning`, rather just has `INFO:root:foo` in it.